tools/logs_agent_QA/testBuilder.py

```diff
@@ -0,0 +1,47 @@
+from enum import Enum
+
+
+class Platform(Enum):
+    linux = 0
+    windows = 1
+    mac = 2
+
+
+class LinuxConfig:
+    platform = Platform.linux
+
+
+class WindowsConfig:
+    platform = Platform.windows
+
+
+class MacConfig:
+    platform = Platform.mac
+
+
+class Suite:
+    def __init__(self, config, tests):
+        self.config = config
+        self.tests = tests
+
+    def build(self, renderDelegate):
+        for test in self.tests:
+            t = test()
+            renderDelegate(t.name, t.render(self.config))
+
+
+class TestCase:
+    name = ""
+
+    def __init__(self):
+        self.steps = []
+
+    def step(self, body):
+        self.steps.append(body)
```

Comment:
It seems like, in practice, these aren't steps so much as chunks of markdown.  Maybe change `step` to `append`?

Reply:
that's fair. 
originally I was experimenting with automatic numbering and some implicit formatting around the `step` function. but ended up removing it as it made formatting harder overall (and markdown is simple enough as it is).